pkg/java/pom/parse.go

```diff
@@ -173,7 +171,7 @@ func (p *parser) parseRoot(root artifact) ([]types.Library, []types.Dependency,
 			// Override the version
 			uniqArtifacts[art.Name()] = artifact{
 				Version:  art.Version,
-				Licenses: art.Licenses,
+				Licenses: append(art.Licenses, result.artifact.Licenses...),
```

Comment:
`org.Licenses` contains licenses from local poms that are not in the maven repository. But I'm not really sure if the user needs these licenses?

Reply:
Where is `art.Licenses` populated? I didn't find it. The root module could have non-empty licenses, but the value must be the same as `result.artifact.Licenses`. Is there any case where using `art.Licenses` is valuable?

Reply:
`org.Licenses` is filled in for root and for other local pos. `result.artifact.Licenses` contains licenses only for poms that are located in the local repository or in the maven repository and if there is no local pom there, then the list of licenses will be empty.

Reply:
> other local POMs

Again, where is it populated?

Reply:
Yes, I know about the root artifact. You said, "other local POMs". Where is it?

>could it be different from result.artifact.Licenses? 

Also, this is my another question. If I understand correctly, `art.Licenses` and `result.artifact.Licenses` are always the same for the root artifact. It doesn't make sense to join them. Please give me an example of the case where we need to join them.

Reply:
Other local POMs are modules or parent POMs.
I said above that `art.Licenses` and `result.artifact.Licenses` will not be the same for the root pom (or any other local one) if it is not in the maven repository, since `result.artifact` is an artifact obtained from the repository.

Reply:
As a result, if we use only `result.artifact.Licenses`, then we will clean up the licenses of local POMs.

Reply:
I think I missed the moment with the cache, I apologize for confusing you

Reply:
No problem. We should add a test case, and then we can see how it works. It may work differently than expected. @DmitriyLewen usually adds a test so that we can easily understand why we need the logic.

Reply:
It seems to me now the tests cover all cases